qcodes/instrument_drivers/QDev/QDac_channels.py

```diff
@@ -251,11 +265,15 @@ def __init__(self,
         self._update_currents = update_currents
         log.info('[+] Done')
 
-    def snapshot_base(self, update=False, params_to_skip_update=None):
-        update_currents = self._update_currents and update
+    def snapshot_base(
+            self,
+            update: Optional[bool] = False,
+            params_to_skip_update: Optional[Sequence[str]] = None
+    ) -> Dict:
+        update_currents = self._update_currents and update is True
         if update:
             self._update_cache(readcurrents=update_currents)
-        self._get_status_performed = True
+            self._get_status_performed = True
```

Comment:
this is now correct with the indentation, right?

Reply:
Yes, the intention is that the status call is only done if update is true. This will enable you correctly update a single parameter if its marked as invalid with a regular snapshot call